spyder_terminal/api.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) Spyder Project Contributors
+#
+# Licensed under the terms of the MIT License
+# (see LICENSE.txt for details)
+# -----------------------------------------------------------------------------
+"""Spyder Terminal Plugin."""
+
+
+class TerminalContainerWidgetActions:
+    NewTerminal = 'new_terminal_action'
+    NewTerminalCWD = 'new_terminal_cwd_action'
+    NewTerminalProject = 'new_terminal_project_action'
+    NewTerminalFile = 'new_terminal_file_action'
```

Comment:
For clarity, let's rename these actions to:

* NewTerminalForCWD
* NewTerminalForProject
* NewTerminalForFile